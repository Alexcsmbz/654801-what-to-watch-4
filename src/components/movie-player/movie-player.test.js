import MoviePlayer from './movie-player.tsx';

const movie = {
  name: `name`,
  genre: `genre`,
  releaseDate: `releaseDate`,
  promo: `promo`,
  poster: `poster`,
  previewMp4: `.mp4`,
  previewWebm: `.webm`
};

describe(`MoviePlayer snapshot test`, () => {
  it(`MoviePlayer should render movie info`, () => {
    const tree = renderer.create(
        <MoviePlayer movie={movie} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
