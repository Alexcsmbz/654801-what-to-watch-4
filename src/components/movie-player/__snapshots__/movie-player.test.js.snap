// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlayer snapshot test MoviePlayer should render movie info 1`] = `
<video
  className="movie-player"
  controls={true}
  height="175"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  poster="poster"
  width="280"
>
  <source
    src=".webm"
    type="video/webm"
  />
  <source
    src=".mp4"
    type="video/mp4"
  />
</video>
`;
